src/main/java/com/ecommerce/ECommerceApp/model/Address.java

```diff
@@ -1,5 +1,9 @@
 package com.ecommerce.ECommerceApp.model;
 
+/**
+ * Represents street, city, state, zip code, and country.
+ * Utilized for mailing & shipping information in the ECommerce system.
+ */
```

Comment:
Added missing Javadoc to help clarify purpose of code, keeping it maintainable and readable.

Source: https://www.tutorialspoint.com/java/java_documentation.htm